cw_monero/lib/monero_wallet.dart

```diff
@@ -729,9 +725,8 @@ abstract class MoneroWalletBase extends WalletBase<MoneroBalance,
 
   void _askForUpdateBalance() {
     final unlockedBalance = _getUnlockedBalance();
-    final fullBalance = _getFullBalance();
-    final frozenBalance = _getFrozenBalance();
-
+    final fullBalance = _getUnlockedBalance() + _getFrozenBalance();
+    final frozenBalance = 0; // this is calculated on the monero side now
```

Comment:
why this change? I don't think it's correct, and it will change the balance with the `frozenBalance` variable, which would be 0

Reply:
Otherwise balance will be incorrect, monero is calculating the frozen balance now